documentation/known_issues.markdown

```diff
@@ -9,9 +9,13 @@ For a list of all known issues, visit our [Issue Tracker](https://tickets.puppet
 
 Here are a few specific issues that we're aware of that might affect certain users:
 
-## Server-side Ruby gems might need to be updated for upgrading from JRuby 1.7
+## puppetserver 6.0.0 dies with `java.lang.OutOfMemoryError: Metaspace`
```

Comment:
Can we maybe break this into two sections? They are both related to JRuby 9k, but the two pieces of advice (gem pathing vs. `ReservedCodeCache`) are sort of unrelated.

Reply:
The codecache info is already below in "## Potential JAVA ARGS settings". Which is maybe a bad title for it. It might be nice to also call out that these are *broken* things that are ever going to be fixed, they're just gotchas in changing requirements that might trip folks up with upgrading.

Reply:
do you mean "never going to be fixed" Justin?

Reply:
Actually, this looks like the revision of this section that was added in 84e77e1 and has been subsequently updated. I think this PR is pulling in an older version of the file?